unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/EntityInformationController.cs

```diff
@@ -0,0 +1,201 @@
+using DCL;
+using DCL.Controllers;
+using DCL.Models;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class EntityInformationController : MonoBehaviour
+{
+    [Header("Prefab references")]
+    public TextMeshProUGUI titleTxt;
+    public TMP_InputField nameIF;
+    public RawImage entitytTumbailImg; 
+    public AttributeXYZ positionAttribute;
+    public AttributeXYZ rotationAttribute;
+    public AttributeXYZ scaleAttribute;
+    public GameObject detailsGO;
+    public GameObject basicsGO;
+    public RectTransform detailsBtn;
+    public RectTransform basicBtn;
+
+    public event Action<Vector3> OnPositionChange;
+    public event Action<Vector3> OnRotationChange;
+    public event Action<Vector3> OnScaleChange;
+
+    public event Action<DCLBuilderInWorldEntity> OnNameChange;
+
+    DCLBuilderInWorldEntity currentEntity;
+    ParcelScene parcelScene;
+
+    bool isEnable = false;
+    bool isChangingName = false;
+
+    int framesBetweenUpdate = 5;
+    int framesCount = 0;
+
+    string loadedThumbnailURL;
+
+    AssetPromise_Texture loadedThumbnailPromise;
+
+    private void Start()
+    {
+        positionAttribute.OnChanged += (x) => OnPositionChange?.Invoke(x);
+        rotationAttribute.OnChanged += (x) => OnRotationChange?.Invoke(x);
+        scaleAttribute.OnChanged += (x) => OnScaleChange?.Invoke(x);
+    }
+
+    private void LateUpdate()
+    {
+        if (!isEnable)
+            return;
+
+        if (currentEntity == null)
+            return;
+
+        if (framesCount >= framesBetweenUpdate)
+        {
+            UpdateInfo(currentEntity);
+            framesCount = 0;
+        }
+        else
+        {
+            framesCount++;
+        }
+    }
+
+    public void ToggleDetailsInfo()
+    {
+        detailsGO.SetActive(!detailsGO.activeSelf);
+        Vector3 angle = detailsBtn.rotation.eulerAngles;
+        angle.z += 180;
```

Comment:
What's this for? If this is for inverting some arrow or something like that, I think we should handle it better (by toggling on some object and off another). 

With this approach, Romi or any UI designer is going to be unable to edit the visuals without modifying the code.